sematic/db/models/external_resource_record.py

```diff
@@ -0,0 +1,168 @@
+# Standard Library
+from typing import Any, Dict, Tuple, Type, Union
+
+# Third-party
+from sqlalchemy import Column, types
+from sqlalchemy.orm import validates
+
+# Sematic
+from sematic.db.models.base import Base
+from sematic.db.models.json_encodable_mixin import JSONEncodableMixin
+from sematic.external_resource import ExternalResource, ResourceState
+from sematic.types.serialization import (
+    type_from_json_encodable,
+    type_to_json_encodable,
+    value_from_json_encodable,
+    value_to_json_encodable,
+)
+
+TypeSerialization = Dict[str, Any]
+
+
+class ExternalResourceRecord(Base, JSONEncodableMixin):
```

Comment:
I know we have another `ExternalResource` object somewhere else, but... can we still keep this named `ExternalResource`?
I believe one is client-side vs. this one which is server-side?

Reply:
sure. In files referring to both, I will import the db one aliased as ExternalResourceRecord